docs/data/base/components/switch/switch.md

```diff
@@ -9,60 +9,72 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/switch/
 
 # Switch
 
-<p class="description">Switches are UI elements that let users choose between two states—most commonly on/off.</p>
+<p class="description">Switch is a UI element that let users choose between two states.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
 {{"component": "modules/components/ComponentPageTabs.js"}}
 
-## Introduction
+{{"demo": "UnstyledSwitchIntroduction", "defaultCodeOpen": false, "bg": "gradient"}}
 
-The Switch component provides users with a switch for toggling between two mutually exclusive states.
+## Installation
 
-{{"demo": "UnstyledSwitchIntroduction", "defaultCodeOpen": false, "bg": "gradient"}}
+Base UI components are all available as a single package.
 
-## Component
+<codeblock storageKey="package-manager">
 
-```jsx
+```bash npm
+npm install @base_ui/react
+```
+
+```bash yarn
+yarn add @base_ui/react
+```
+
+```bash pnpm
+pnpm add @base_ui/react
+```
+
+</codeblock>
+
+Once you have the package installed, import the component.
+
+```ts
 import * as Switch from '@base_ui/react/Switch';
 ```
 
-### Anatomy
+## Anatomy
```

Comment:
I think we miss the equivalent of [these section](https://deploy-preview-392--base-ui.netlify.app/base-ui/react-number-field/#value)

Same for Checkbox

Reply:
Will consider for a separate PR.